literature/readme.md

```diff
@@ -6,4 +6,7 @@
 
 
 ### Project Notes
-Please visit [Zotero](INSERT YOUR LINK HERE) to see relevant publications reviewed.
\ No newline at end of file
+Please visit [Zotero](INSERT YOUR LINK HERE) to see relevant publications reviewed.
+
+Currently, the literature review for this project can be found here:
+https://drive.google.com/drive/folders/1J0M274KKq_RivIs6aHqdbdISlRyDWF5z
```

Comment:
@SarahMalykke I think you can add the Zotero link (even if it's just a work in progress).  For putt-putt-dataset, the weblink to the annotated folder is this:
https://www.zotero.org/groups/2934532/ndc_lab/collections/B3WYU6GK